lib/core/application/backend.ts

```diff
@@ -385,10 +385,12 @@ class InternalLogger extends ApplicationManager implements InternalLogger {
 
   private _log (level: string, message: any) {
     if (! this._application.started) {
-      throw runtimeError.get('unavailable_before_start', 'log');
+      // eslint-disable-next-line no-console
+      console.log(util.inspect(message));
+    }
+    else {
+      this._kuzzle.log[level](util.inspect(message));
     }
```

Comment:
Unnecessary: kuzzle's `log` function already does that.
See https://github.com/kuzzleio/kuzzle/blob/master/lib/kuzzle/log.js

Reply:
Yes but Kuzzle is not even instantiated before calling `Backend.start`, and I can't instantiate Kuzzle before that because Kuzzle take the configuration in the constructor so the custom configuration will not be loaded correctly:

```
// Kuzzle is instantiated when Backend is instantiated so we can use the logger
const app = new Backend('test')

app.log.info('it works')

// Kuzzle has already have been instantiated with the configuration
app.config.set('server.port', 4242)

// Kuzzle will listen on 7512
app.start()
```

Reply:
ah, yes, of course :sweat_smile: